modules/custom/falcon_development/config/install/features.bundle.falcon_feature.yml

```diff
@@ -11,7 +11,7 @@ assignments:
   alter:
     core: true
     uuid: true
-    user_permissions: true
+    user_permissions: false
```

Comment:
Out of curiosity, why did we disable it?

Reply:
By default features do not export user roles permissions (cleans them up from user role configs)... setting it to `false` allows to export permissions with roles.